vms/platformvm/service.go

```diff
@@ -973,7 +979,9 @@ func (s *Service) GetPendingValidators(_ *http.Request, args *GetPendingValidato
 
 	// Create set of nodeIDs
 	nodeIDs := set.Set[ids.NodeID]{}
-	nodeIDs.Add(args.NodeIDs...)
+	for _, nodeID := range args.NodeIDs {
+		nodeIDs.Add(nodeID)
+	}
```

Comment:
I think we should revert this change (or maybe use `set.Of`?)

Reply:
Done